pkg/refactor/database/db_schema.sql

```diff
@@ -178,33 +178,41 @@ CREATE TABLE IF NOT EXISTS "staging_events" (
     "type" text NOT NULL,
     "cloudevent" text NOT NULL,
     "namespace_id" uuid NOT NULL,
+    "namespace" text,
     "namespace_name" text,
     "received_at" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
     "created_at" timestamptz NOT NULL,
     "delayed_until" timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
     FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("id") ON DELETE CASCADE ON UPDATE CASCADE,
+--    FOREIGN KEY ("namespace") REFERENCES "namespaces"("name") ON DELETE CASCADE ON UPDATE CASCADE
     CONSTRAINT "no_dup_stag_check" UNIQUE ("source","event_id", "namespace_id"),
     PRIMARY KEY ("id")
 );
 
 CREATE TABLE IF NOT EXISTS "events_history" (
```

Comment:
What is this for? Why do we need another ID field? Is it because a cloudevent ID might not be unique if it hits multiple namespaces? Please add a code comment here.